src/stores/FrameStore.ts

```diff
@@ -435,17 +435,28 @@ export class FrameStore {
     }
 
     @computed get positionAxis(): number {
+        return this.specialAxis(/offset|position|offset position/i);
+    }
+
+    @computed get isUVImage(): boolean {
+        return this.uvAxis !== undefined;
+    }
+
+    @computed get uvAxis(): number {
```

Comment:
I don't think this should be a `@computed`, should it? since nothing here is an observable.

Same goes for the above two functions

Reply:
Refactored this part so that the values will be updated correctly.

Reply:
Perhaps you misunderstood me. Having it as a get function is fine, but the '@computed' decorator is only useful if your get function reads observables that change

Reply:
I thought the @computed decorator detects observables in functions, but then I realized that it does not and refactored this part. I think the @computed decorator is needed here.

Reply:
to be dealt with in #1402